packages/teleport/src/DesktopSession/ActionMenu.tsx

```diff
@@ -41,6 +47,9 @@ export default function ActionMenu(props: Props) {
 }
 
 type Props = {
+  canShareDirectory: boolean;
+  isSharingDirectory: boolean;
+  onShareDirectory: VoidFunction;
```

Comment:
As these three props aren't ever used in isolation I feel like you can simplify this logic by combining the two boolean values into one `shouldShowDirectoryShare` and then pass in the callback as well.